extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -79,7 +79,19 @@ function render_block( $attributes ) {
 		return '<p>' . esc_html( $player_data->get_error_message() ) . '</p>';
 	}
 
-	return render_player( $player_data, $attributes );
+	/*
+	 * Get the block attributes checking if `$attributes`
+	 * is an array and it has defined a property.
+	 *
+	 * It handles the dual-behavior of argument
+	 * of the callback_render() function, which
+	 * was recently introduced by this Pull Request:
+	 * https://github.com/WordPress/gutenberg/pull/21467
+	 */
+	$block_attributes = ! is_array( $attributes ) && isset( $attributes->attributes )
+		? $attributes->attributes
+		: $attributes;
```

Comment:
Would this work?

```suggestion
	if ( ! empty( $attributes->attributes ) ) {
		$attributes = $attributes->attributes;
	}
```

That way you'd just provide forward compatibility and not have to change the next line.

Reply:
yes, right. I wouldn't like to use the same reference but I think it's ok.